qiskit/passmanager/passmanager.py

```diff
@@ -47,6 +47,8 @@ def __init__(
         """
         self._tasks = []
         self.max_iteration = max_iteration
+        # This empty property set never gets used; it gets overridden at the completion of a
```

Comment:
Do you want to nullify this attribute? This used to be `None` in < 0.45. This update disallows users to manually update property_set (e.g. warm-start), but I think this is reasonable. This constructor may mislead the users if they don't read the comment.

Reply:
Oh also, given that it was `PropertySet` in 1.0.0, we can't backport the PR if we change the dummy property set to `None`.  That actually means we wouldn't be able to accept that change to this PR, so maybe if you want to talk about it more, we should make a new issue for Qiskit 1.1.